master/getting-started/docker/tutorials/basic.md

```diff
@@ -215,6 +214,14 @@ docker network create --driver calico --ipam-driver calico-ipam --subnet=192.0.2
 docker run --net my_net --name my_workload --ip 192.0.2.100 -tid busybox
 ```
 
+Verify that the IP address was assigned to the container by running:
+
+```
+docker inspect my_workload | grep IPAddress
```

Comment:
Nope, use the --format option to return a single field value.  Look at the command above for inspiration.